o2a/converter/oozie_converter.py

```diff
@@ -121,6 +128,47 @@ def convert_dependencies(workflow: Workflow) -> None:
         for node in workflow.nodes.values():
             workflow.dependencies.update(node.mapper.required_imports())
 
+    @staticmethod
+    def convert_relations(workflow: Workflow) -> None:
+        """
+        Given a dictionary of task_ids and ParsedActionNodes,
+        returns a set of logical connectives for each task in Airflow.
+
+        :return: Set with strings of task's downstream nodes.
+        """
+        logging.info("Parsing relations between operators.")
+        for p_node in workflow.nodes.values():
+            for downstream in p_node.get_downstreams():
+                relation = Relation(
+                    from_task_id=p_node.last_task_id, to_task_id=workflow.nodes[downstream].first_task_id
+                )
+                workflow.relations.add(relation)
+            error_downstream = p_node.get_error_downstream_name()
+            if error_downstream:
+                relation = Relation(
+                    from_task_id=p_node.last_task_id,
+                    to_task_id=workflow.nodes[error_downstream].first_task_id,
+                )
+                workflow.relations.add(relation)
+
+    @staticmethod
+    def update_trigger_rules(workflow: Workflow) -> None:
+        """
+        Updates the trigger rules of each node based on the downstream and
+        error nodes.
+        """
+        for node in workflow.nodes.values():
+            # If a task is referenced  by an "ok to=<task>", flip bit in parsed
```

Comment:
This comment is wrong. I think it should be removed as the code is clear enough that it does not need extra explanation.

Reply:
I updated the PR. Thanks.